### ISSUE TITLE
ListItemModel not exported from '@progress/kendo-angular-buttons'

### ISSUE THREAD

**Issue by SzymonKatra** [telerik/kendo-angular#2151]
**Describe the bug**
`ListItemModel` is not exported from the `@progress/kendo-angular-buttons`

`ERROR in src/app/app.component.ts(2,26): error TS2305: Module '"~/myapp/node_modules/@progress/kendo-angular-buttons/dist/es2015/index"' has no exported member 'ListItemModel'.`

The only way to import is to use
`import { ListItemModel } from '@progress/kendo-angular-buttons/dist/es2015/listbutton/list-item-model';`

**To Reproduce**
https://stackblitz.com/edit/angular-tg4geq
- Download example
- `npm install` & `ng build`

**Expected behavior**
`ListItemModel` should be exported from the `@progress/kendo-angular-buttons` package.

**Environment**
`"@progress/kendo-angular-buttons": "4.3.3"`

```"core-js": "2.5.7",
"zone.js": "0.8.26",
"@angular/core": "7.2.10",
"@angular/forms": "7.2.10",
"@angular/common": "7.2.10",
"@angular/router": "7.2.10",
"@angular/compiler": "7.2.10",
"@angular/platform-browser": "7.2.10",
"@angular/platform-browser-dynamic": "7.2.10",
"rxjs-compat": "6.3.3",
"web-animations-js": "2.3.1",
"@angular/animations": "7.2.10",
"@angular/http": "7.2.10",
"hammerjs": "2.0.8",
"@telerik/kendo-intl": "1.4.4",
"@progress/kendo-date-math": "1.4.0",
"@progress/kendo-drawing": "1.5.9",
"@progress/kendo-data-query": "1.5.1",
"@progress/kendo-file-saver": "1.0.7",
"@progress/kendo-charts": "1.13.0",
"@progress/kendo-recurrence": "0.1.2",
"@progress/kendo-angular-buttons": "4.3.3",
"@progress/kendo-angular-charts": "3.8.0",
"@progress/kendo-angular-conversational-ui": "1.1.1",
"@progress/kendo-angular-dateinputs": "3.7.1",
"@progress/kendo-angular-dialog": "3.11.0",
"@progress/kendo-angular-dropdowns": "3.5.2",
"@progress/kendo-angular-excel-export": "2.3.0",
"@progress/kendo-angular-gauges": "2.4.1",
"@progress/kendo-angular-grid": "3.13.1",
"@progress/kendo-angular-inputs": "4.2.1",
"@progress/kendo-angular-intl": "1.7.0",
"@progress/kendo-angular-l10n": "1.3.0",
"@progress/kendo-angular-label": "1.3.0",
"@progress/kendo-angular-layout": "3.3.0",
"@progress/kendo-angular-menu": "1.2.0",
"@progress/kendo-angular-notification": "1.2.0",
"@progress/kendo-angular-pdf-export": "1.2.0",
"@progress/kendo-angular-popup": "2.5.0",
"@progress/kendo-angular-resize-sensor": "3.1.2",
"@progress/kendo-angular-ripple": "1.1.1",
"@progress/kendo-angular-scheduler": "0.10.1",
"@progress/kendo-angular-scrollview": "2.1.0",
"@progress/kendo-angular-sortable": "2.0.1",
"@progress/kendo-angular-toolbar": "1.1.0",
"@progress/kendo-angular-editor": "0.3.0",
"@progress/kendo-angular-tooltip": "1.2.0",
"@progress/kendo-angular-treeview": "3.1.0",
"@progress/kendo-angular-upload": "4.2.2"```
```

**Comment by ipeshev** [comment]
Fix available in [latest dev version](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/).

**Comment by SzymonKatra** [comment]
Works fine now, thanks.

**Comment by tsvetomir** [comment]
Shipped in v4.4.0
